src/pages/news/news-item/news-item.style.js

```diff
@@ -45,13 +45,16 @@ export const useStyles = makeStyles(() => ({
   },
   newsItemContent: {
     padding: '0px',
-    height: '270px',
-    overflow: 'hidden'
+    overflow: 'hidden',
+    paddingButtom: '50px',
```

Comment:
it should be paddingBottom

Reply:
Fixed (deleted because it's not necessary)